src/screens/ChannelListScreenV2/AnonymousChannelListScreen/index.tsx

```diff
@@ -13,7 +13,9 @@ import {ANONYMOUS, ANON_PM, ANON_POST_NOTIFICATION} from '../../../hooks/core/co
 import useAnonymousChannelListScreenHook from '../../../hooks/screen/useAnonymousChannelListHook';
 import useRootChannelListHook from '../../../hooks/screen/useRootChannelListHook';
 import Search from '../../ChannelListScreen/elements/Search';
+import {COLORS} from '../../../utils/theme';
 import IncognitoEmptyChat from '../IncognitoEmptyChat';
+import {COLORS} from '../../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There is a redundant import statement for `COLORS` from `'../../../utils/theme'`. This import is declared twice which is unnecessary and can lead to confusion.
```diff
- import {COLORS} from '../../../utils/theme';
  import IncognitoEmptyChat from '../IncognitoEmptyChat';
- import {COLORS} from '../../../utils/theme';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->